ocaml-lsp-server/src/document.ml

```diff
@@ -222,14 +222,15 @@ let make wheel config pipeline (doc : DidOpenTextDocumentParams.t)
 
 let update_text ?version t changes =
   match Text_document.apply_content_changes ?version (tdoc t) changes with
-  | exception Text_document.Invalid_utf8 ->
+  | exception Text_document.Invalid_utf message ->
     Log.log ~section:"warning" (fun () ->
         Log.msg
           "dropping update due to invalid utf8"
           [ ( "changes"
             , Json.yojson_of_list
                 TextDocumentContentChangeEvent.yojson_of_t
                 changes )
+          ; ("message", `String message)
```

Comment:
I wonder whether this log message could be a bit more helpful if we reported the context around the exception - on what document, after/on which changeEvent and with what doc state the error occurred